manifests/record.pp

```diff
@@ -44,7 +44,7 @@
   validate_hash($hash_data)
 
   $records_template = $content_template ?{
-    ''      => 'bind/default-record.erb',
+    undef   => 'bind/default-record.erb',
```

Comment:
Could maybe be improved by setting `$content_template = 'bind/default-record.erb'` in class parameters and using `content_template` instead of `records_template`.